index.html

```diff
@@ -519,6 +519,80 @@ <h3>LinkedDataSignature2016</h3>
           </pre>
       </section>
 
+
+      <section id="MerkleProof2019" about="https://w3id.org/security#MerkleProof2019">
+
+        <h3>MerkleProof2019</h3>
+          <p>
+A Linked Data signature is used for digital signatures on RDF Datasets.
+The default canonicalization mechanism is specified in the RDF Dataset
+Normalization specification, which effectively deterministically names all
+unnamed nodes. The default signature mechanism
+uses a SHA-256 digest and ECDSA to perform the digital signature. More information at
+              <a href="https://w3c-ccg.github.io/lds-merkle-proof-2019/">https://w3c-ccg.github.io/lds-merkle-proof-2019/</a>
+          </p>
+          <dl>
+            <dt>Status</dt>
+            <dd property="vs:term_status">stable</dd>
+            <dt>Parent Class</dt>
+            <dd>LinkedDataSignature</dd>
+            <dt>Expected properties</dt>
+            <dd>proofValue</dd>
+            <dt>Signature Properties</dt>
+            <dd>
+              <dl>
+                <dt>Default Canonicalization Algorithm</dt>
+                <dd>
+                    <a rel="canonicalizationAlgorithm"
+                       href="https://w3id.org/rdf#URDNA2015">
+                        https://w3id.org/rdf#URDNA2015</a>
+                </dd>
+                <dt>Default Signature Algorithm</dt>
+                <dd>
+<a rel="signatureAlgorithm"
+  href="http://www.w3.org/2000/09/xmldsig#rsa-sha256">
+http://www.w3.org/2000/09/xmldsig#rsa-sha256</a>
```

Comment:
```suggestion
  href="http://www.w3.org/2001/04/xmldsig-more#ecdsa-sha256">
http://www.w3.org/2001/04/xmldsig-more#ecdsa-sha256</a>
```

Reply:
@msporny we had a quick discussion with @dsemenovsky and we are not sure what the actual issue is. 
SHA-256 is used as the canonicalization algorithm of the document, and ECDSA is the signature of the blockchain transaction. 

MerkleProof2019 is specifically designed to work with Blockchains since its anchoring is used as part of the CBOR encoded `proofValue`. You may see here an example of what is encoded into the `proofValue`. https://w3c-ccg.github.io/lds-merkle-proof-2019/#example-1

Now that also raises a question on my side, should we make mention of CBOR anywhere in this addition? If so, under what definition item should it be listed?

Reply:
No need to mention CBOR anywhere in this addition since the MerkleProof2019 specification defines what it should be. So there is nothing more to do in this specific PR.

Thanks for the link to the example... some thoughts on MerkleProof2019 -- Example 1 is a little strange, in that the text encoding seems to be hex where the hashing algorithm is assumed to be SHA-256? I suggest that you encode the text example as a multibase encoded multihash value. This will make it so that the data self-identifies as a base-16 encoded sha-256 32 byte value (for example). That will make MerkleProof2019 completely forwards compatible if you ever have to change the base-encoding format (to base58/base64url, for example) or the hash format (to SHA-3, etc.). 

Just some food for thought.